main.js

```diff
@@ -1,4 +1,4 @@
-var scope = (typeof self !== "undefined" && self) || window;
+var scope = this;
```

Comment:
Hmm, per #34 the goal seems to be to support Webpack wrapping: `(function(global) { … })();`

So how about `var scope = typeof global !== "undefined" ? global : (typeof self !== "undefined" ? self : window);`?

Reply:
Never mind, Webpack correctly preserves the global `this` by doing `.call(this)`. So the current patch is fine.